permissions/pom-bom.yml

```
@@ -2,6 +2,10 @@
 name: "bom"
 github: "jenkinsci/bom"
 paths:
-- "io/jenkins/tools/bom/bom"
+- "io/jenkins/tools/bom/bom-2.190.x"
```

Could be reduced to so you do not have to repeat requesting permission for new Jenkins versions.
```suggestion
- "io/jenkins/tools/bom"
```